lib/Basics/ReadWriteLock.h

```diff
@@ -89,13 +90,13 @@ class ReadWriteLock {
   /// the last 16 bits the number of active readers.  
   std::atomic<uint32_t> _state;
 
-  static const unsigned WRITE_LOCK = 1;
+  static constexpr uint32_t WRITE_LOCK = 1;
```

Comment:
can you remove this change from this PR please, and open a separate PR just for the lock code changes?